src/makeCalculator.js

```diff
@@ -4,7 +4,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculatorObject = {
+    result: 0,
+    operate(operator, value) {
+      this.result = operator(this.result, value);
+
+      return this;
+    },
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      return a / b;
```

Comment:
Consider implementing a check to avoid division by 0

Reply:
@anastasiiavorobiova done ✅